### PR TITLE
First crack a web interface

### PR DESCRIPTION
**Pull request by wittjosiah**

### CODE REVIEW

File: web/src/Stylesheets.elm [hunk 1]
```diff
@@ -12,7 +25,13 @@ port files : CssFileStructure -> Cmd msg
 fileStructure : CssFileStructure
 fileStructure =
     Css.File.toFileStructure
-        [ ( "../../stylesheets/main.css", Css.File.compile [ css ] ) ]
+        [ ( "../../stylesheets/main.css"
+          , Css.File.compile
+                [ generalCss
```
Comment (kiambogo) [inline web/src/Stylesheets.elm]: Too many spaces here

Comment (kiambogo) [inline web/src/Stylesheets.elm:30]: Too many spaces here

**Review by kiambogo — APPROVED** [review]
Nit with spacing, but otherwise LGTM

File: web/src/Stylesheets.elm (hunk 1 above)
Comment (wittjosiah) [reply web/src/Stylesheets.elm:30]: it does look like there's one or two extra spaces in there, but that's how elm-format saves it so you'll have to find the bug in elm-format if you want to change it 😏

**Pull request merged**